examples/multiclass/plot_multiclass_overview.py

```diff
@@ -0,0 +1,198 @@
+:mod:`sklearn.multiclass` module implements various strategies that
+one can use for experimenting. This example will review them.
+
+One should not confuse the OvO/OvR strategy used for the evaluation of
+multiclass classifiers with the OvO/OvR strategy used to train a multiclass
+classifier by fitting a set of binary classifiers (the
+:class:`~sklearn.multiclass.OneVsOneClassifier` and
+:class:`~sklearn.multiclass.OneVsRestClassifier` meta-estimator).
+"""
+
+# %%
+# The Yeast UCI dataset
+# ---------------------
+#
+# In this example, we use a UCI dataset [1]_, generally refered as the Yeast
+# dataset. We use the :func:`sklearn.datasets.fetch_openml` function to load
+# the dataset from OpenML.
+from sklearn.datasets import fetch_openml
+
+X, y = fetch_openml(data_id=181, as_frame=True, return_X_y=True, parser="pandas")
+
+# %%
+# To know the type of data science problem we are dealing with, we can check
+# the target for which we want to buid a predictive model.
+y.value_counts().sort_index()
+
+# %%
+# We see that the target is discrete and composed of 10 classes. We therefore
+# deal with a multiclass classification problem.
+#
+# Strategies comparison
+# ---------------------
+#
+# In the following experiment, we use a
+# :class:`~sklearn.tree.DecisionTreeClassifier` and a
+# :class:`~sklearn.model_selection.RepeatedStratifiedKFold` cross-validation
+# with 5 splits and 10 repetitions.
+#
+# We compare the following strategies:
+#
+# * :class:`~sklearn.tree.DecisionTreeClassifier` which supports multiclass
+#   classification by default. The sequential partitioning of the training
+#   dataset isolates data point and affect to the most frequent class
+#   present in the partition. Thus, the decision tree is able to deal with
+#   multiclass classification.
+# * :class:`~sklearn.multiclass.OneVsOneClassifier` trains a set of binary
+#   classifiers where each classifier is trained to distinguish between
+#   two classes.
+# * :class:`~sklearn.multiclass.OneVsRestClassifier`: trains a set of binary
+#   classifiers where each classifier is trained to distinguish between
+#   one class and the rest of the classes.
+# * :class:`~sklearn.multiclass.OutputCodeClassifier`: trains a set of binary
+#   classifiers where each classifier is trained to distinguish between
+#   a set of classes from the rest of the classes. The set of classes is
+#   defined by a codebook, which is randomly generated in scikit-learn. This
+#   method exposes a parameter `code_size` to control the size of the codebook.
+#   We set it above one since we are not interested in compressing the class
+#   representation.
+import pandas as pd
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.multiclass import (
+    OneVsOneClassifier,
+    OneVsRestClassifier,
+    OutputCodeClassifier,
+)
+from sklearn.model_selection import cross_validate, RepeatedStratifiedKFold
+
+cv = RepeatedStratifiedKFold(n_splits=5, n_repeats=10, random_state=0)
```

Comment:
I reduced the number of splits and repeat but with the care that the figures are the same.
Let's see if I speed up the example enough. It is weird because it takes a couple of seconds on my laptop :).